### DISCUSSION TITLE
Crop yield estimation

### DISCUSSION THREAD

**Discussion by Chun-HNUST** [EDmodel/ED2#386] [Q&A] [answered]
Can ED model be used to estimate crop yields? For example, rice yield estimation.

**Comment by xiangtaoxu** [comment, marked as answer]
I am not aware of any study using ED2 to model crop yield, in part because ED models were designed to represent heterogeneity in forests while crop lands are relatively homogeneous. @mdietze had some [paper](https://esajournals.onlinelibrary.wiley.com/doi/full/10.1890/12-0854.1?casa_token=JrS0LjfX7HUAAAAA%3ANpMeIy0n-XmasQSJ9rsQ52fqHVrYoirhjJBc6Kgsr_0KQQTd5r3lbALERwGIreNa9dPzSXgxRs_hK7E) to model poplar plantation.

It is definitely possible to model rice yield if you adapt some grass PFTs in the model to represent rice. Meanwhile, there are a suite of agriculture models (e.g. APSIM, Ecosys) specially designed and parameterized for crops.

**Comment by Chun-HNUST** [comment]
Thank you very much! This is very helpful to me.
